index.html

```diff
@@ -3750,6 +3777,180 @@ <h2>
           </tr>
         </table>
       </section>
+      <section data-dfn-for="MerchantValidationEvent" data-link-for=
+      "MerchantValidationEvent">
+        <h2>
+          <dfn>MerchantValidationEvent</dfn> interface
+        </h2>
+        <pre class="idl">
+          [Constructor(DOMString type, optional MerchantValidationEventInit eventInitDict), SecureContext, Exposed=Window]
+          interface MerchantValidationEvent : Event {
+            readonly attribute USVString validationURL;
+            void complete(Promise&lt;any&gt; merchantSessionPromise);
+          };
+        </pre>
+        <section>
+          <h3>
+            <dfn data-lt=
+            "MerchantValidationEvent.MerchantValidationEvent()"><code>MerchantValidationEvent</code>
+            Constructor</dfn>
+          </h3>
+          <p data-tests=
+          "MerchantValidationEvent/constructor.https.html, MerchantValidationEvent/constructor.http.html">
+            The <a data-lt=
+            "MerchantValidationEvent"><code>MerchantValidationEvent(<var>type</var>,
+            <var>eventInitDict</var>)</code></a> constructor MUST act as
+            follows:
+          </p>
+          <ol class="algorithm">
+            <li>Let <var>base</var> be the <a data-cite=
+            "dom#context-object">context object</a>’s <a data-cite=
+            "!html/multipage/webappapis.html#relevant-settings-object">relevant
```

Comment:
There is no context object in constructors. But once you use event constructing steps you can use _event_'s relevant settings object.